src/e2e/specs/dashboard.spec.ts

```diff
@@ -31,8 +32,67 @@ test.describe("Breaches Dashboard - Headers", () => {
     expect(await dashboardPage.dashboardNavButton.getAttribute("href")).toEqual(
       "/redesign/user/dashboard",
     );
-    expect(await dashboardPage.fAQsNavButton.getAttribute("href")).toEqual(
+    expect(await dashboardPage.FAQsNavButton.getAttribute("href")).toEqual(
       "https://support.mozilla.org/kb/firefox-monitor-faq",
     );
   });
+
+  test("Verify that the site header and navigation bar is displayed correctly", async ({
+    dashboardPage,
+  }) => {
+    // link to testrail
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2301511",
+    });
+
+    // verify the navigation bar left side elements
+    await expect(async () => {
+      expect(
+        await dashboardPage.fireFoxMonitorLogoImgButton.isVisible(),
+      ).toBeTruthy();
+      expect(await dashboardPage.dashboardNavButton.isVisible()).toBeTruthy();
+      expect(await dashboardPage.exposuresHeading.isVisible()).toBeTruthy();
+      expect(await dashboardPage.FAQsNavButton.isVisible()).toBeTruthy();
+    }).toPass();
```

Comment:
Forgot to ask earlier, but what's the reason for grouping these in a wrapper `expect(...).toPass()` vs just nudging the `expect()`s a level higher without the wrapper?

Reply:
its useful for less flakiness when dealing with a path to an assertion but it is not required/necessary here.